src/LosReCaptcha/Service/ReCaptcha.php

```diff
@@ -468,7 +478,14 @@ protected function query($responseField)
         }
 
         /* Fetch an instance of the http client */
-        $httpClient = $this->getHttpClient();
+        $httpClient = new HttpClient(
```

Comment:
Here it's better keep the $this->getHttpClient() so one can inject it's own implementation. For example, this code uses '/etc/ssl/certs', but in a Windows machine won't work. So you can create this HttpClient outside and inject in the service using $service->setHttpClient($client);